carta/session.py

```diff
@@ -388,10 +388,28 @@ def append_image(self, path, hdu="", complex=False, expression=ArithmeticExpress
         complex : {2}
             Whether the image is complex. Set to ``False`` by default.
         expression : {3}
-            Arithmetic expression to use if appending a complex-valued image. The default is :obj:`carta.constants.ArithmeticExpression.AMPLITUDE`.
+            Complex expression to use if appending a complex-valued image. The default is :obj:`carta.constants.ComplexExpression.AMPLITUDE`.
         """
         return Image.new(self, path, hdu, True, complex, expression)
 
+    @validate(String(), String(), String(r"\d*"), Boolean())
+    def load_LEL_image(self, directory, expression, hdu="", append=False):
+        """Open or append a new image via the Lattice Expression Language (LEL) interface.
+
+        Parameters
+        ----------
+        directory : {0}
+            The directory of the image file(s), either relative to the session's current directory or an absolute path relative to the CARTA backend's root directory.
+        expression : {1}
+            The LEL arithmetic expression.
+        hdu : {2}
+            The HDU to select inside the file.
```

Comment:
How is this value used in the frontend if a LEL expression with multiple images is provided? Is the same HDU applied to each image? Is it not applied at all?